MyHome.UI/Helpers/HelperMethods.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace MyHome.UI.Helpers
+{
+    public static class HelperMethods
+    {
+        public static bool IsDouble(this string input)
+        {
+            double dbToParse; 
```

Comment:
If we have changed the amount type to be decimal, this should probably also change